akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowOnCompleteSpec.scala

```diff
@@ -78,6 +78,18 @@ class FlowOnCompleteSpec extends StreamSpec with ScriptedTest {
       onCompleteProbe.expectMsg(Success(Done))
     }
 
+    "yield error on abrupt termination" in assertAllStagesStopped {
+      val mat = ActorMaterializer()
```

Comment:
Oh, thanks for pointing that out. I think I'll just remove the assertAllStages in that case, since I know the materializer is killed.